### PR TITLE
Get map navigation buttons working with `momo-component-map`

### PR DESCRIPTION
**Pull request by annarieger**
Since we use our own `momo-component-map` component in frontend, some adjustments were needed to get map navigation buttons (`StepForward` and `StepBack`) working again.

### CODE REVIEW

**Comment by annarieger** [comment]
could you please review this MR, @marcjansen?

File: app/view/component/MapController.js [hunk 1]
```diff
@@ -251,19 +251,21 @@ Ext.define('momo.view.component.MapController', {
 
     setRouting: function(){
         var mapComponentXType = 'momo-component-map';
+        var viewModel =
+            this.getView().up().down(mapComponentXType).getViewModel();
```
Comment (marcjansen) [inline app/view/component/MapController.js:255]: Did you try `this.getView().getViewModel();`?

**Comment by marcjansen** [comment]
I added one question, regardless of the outcome, please merge.

File: app/view/component/MapController.js (hunk 1 above)
Comment (annarieger) [inline app/view/component/MapController.js:255]: Yes, i tried it. We need to be more specific here to get the right `viewModel` bounded to the `momo-component-map` component. That's why I use here this pretty complex query.

**Comment by annarieger** [comment]
Thx! Will merge now.

**Pull request merged**